includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -684,42 +684,43 @@ public function display_payment_request_button_separator_html() {
 	 */
 	private function is_product_supported() {
 		$product = $this->get_product();
+		$is_supported = true;
```

Comment:
Nitpick: update this to avoid lint error https://github.com/Automattic/woocommerce-payments/runs/4260880320?check_suite_focus=true

```suggestion
		$product      = $this->get_product();
		$is_supported = true;
```

Reply:
Closed by a9a689b